.github/actions/print_logs/action.yaml

```diff
@@ -0,0 +1,65 @@
+name: Print Logs
+description: Processes test results to produce a test summary and upload results to DataDog.
+inputs:
+  logfile_glob:
+    description: "Path (or glob) to test log files."
+    required: true
+  test_status:
+    description: "Test status. If the value is not success this step will fail."
+    required: false
+    default: 'success'
+  symbolize:
```

Comment:
Symbolization support is added but is not enabled. The stack.py script fails on python imports and needs additional setup. Instead of ripping it out of here it will be fixed forward soon.